layers.py

```diff
@@ -935,7 +1005,11 @@ def forward(
             dense_xyz,
             dense_xyz_norm,
             dense_feats=None,
-            vi_features=None):
+            vi_features=None,
+            inv_neighbors=None,
+            inv_k=None,
+            inv_idx=None
+            ):
```

Comment:
Comment of the function should include these 4 variables along with their sizes as well.